core/src/main/java/org/elasticsearch/search/aggregations/bucket/adjacency/AdjacencyMatrixAggregationBuilder.java

```diff
@@ -57,15 +57,15 @@
     private String separator = DEFAULT_SEPARATOR;
 
     public static Aggregator.Parser getParser() {
```

Comment:
to be honest, I don't know the reason why this is constructed the way it is. Will see if I can change this.